packages/http/src/router/__tests__/matchBaseUrl.spec.ts

```diff
@@ -55,8 +56,8 @@ describe('matchServer.ts', () => {
               host: { default: 'www.example.com' },
             },
           },
-          'http://stoplight.io/v1',
-        ),
+          'http://stoplight.io/v1'
+        )
```

Comment:
Did you change this or was prettier reformatting the code?

Reply:
It surely wasn't intentional, I might have accidentally fired some formatting tool on it though. I'm going to check what happens if I put it back and I fire prettier on it, and we'll see.

Reply:
Double checked, running prettier removes them.

Reply:
No that's fine, I know that we upgrader prettier some time ago and it's refactoring the code as we change it.

Reply:
@marcelltoth We should have an hook on TS files that should reformat the source code even if you format in your own way — that's kind of weird.